src/main/java/lotto/Logic/Calculator.java

```diff
@@ -0,0 +1,32 @@
+package lotto.Logic;
+
+import java.util.Map;
+import lotto.Constant.MatchingNumberCount;
+
+public class Calculator {
+    //계산기
+    public static int totalMoney;
+    public static double rateOfReturn;
+
+    private static final double PERCENTAGE = 100;
+
+    public static double calculate(Map<String, Integer> totalResultMap, int budget) {
+        totalMoneyAdd(totalResultMap);
+        return rating(budget);
+    }
+
+    public static double rating(int budget) {
+        rateOfReturn = (double) totalMoney / budget * PERCENTAGE; //
+        rateOfReturn = Math.round(rateOfReturn * PERCENTAGE) / PERCENTAGE;
+        return rateOfReturn;
+    }
+
+    private static void totalMoneyAdd(Map<String, Integer> totalResultMap) {
+        int threePrice = MatchingNumberCount.THREE.getWinningPrices(totalResultMap.get("Three"));
+        int fourPrice = MatchingNumberCount.FOUR.getWinningPrices(totalResultMap.get("Four"));
+        int fivePrice = MatchingNumberCount.FIVE.getWinningPrices(totalResultMap.get("Five"));
+        int five_BonusPrice = MatchingNumberCount.FIVE_BONUS.getWinningPrices(totalResultMap.get("Five_Bonus"));
+        int sixPrice = MatchingNumberCount.SIX.getWinningPrices(totalResultMap.get("Six"));
+        totalMoney = threePrice + fourPrice + fivePrice + five_BonusPrice + sixPrice;
```

Comment:
EnumMap이나 Map의 key를 enum의 name으로 처리해두면 좋을 것 같아요
```suggestion
    private static void calculateTotalMoney(Map<String, Integer> totalResultMap) {
        for (MatchingNumberCount count: MatchingNumberCount.values()) {
           totalMoney += count.getWinningPrices(totalResultMap.get(count));
        }
```